draft-ietf-quic-transport.md

```diff
@@ -1768,9 +1770,9 @@ completes, the client uses the transport parameters established in the
 handshake.  Not all transport parameters are remembered, as some do not apply to
 future connections or they have no effect on use of 0-RTT.
 
-The definition of new transport parameters ({{new-transport-parameters}}) MUST
-specify whether they MUST, MAY, or MUST NOT be stored for 0-RTT. A client need
-not store a transport parameter it cannot process.
+The definition of new transport parameters ({{new-transport-parameters}}) is
+REQUIRED to specify whether they MUST, MAY, or MUST NOT be stored for 0-RTT. A
```

Comment:
```suggestion
The definition of a new transport parameter ({{new-transport-parameters}}) is
REQUIRED to specify whether it MUST, MAY, or MUST NOT be stored for 0-RTT. A
```
I think the singular reads better, especially with the change in keyword.

Reply:
Also, MUST > REQUIRED.

Reply:
No, MUST == REQUIRED. (I think you are thinking of RECOMMENDED?)